README.md

```diff
@@ -249,4 +249,4 @@ This ansible repository deploys MySQL server on OpenStack provider.
 ### Run molecule tests
 
 1. pip install -r test-requirements.txt
-1. molecule -s vault test
+1. molecule test -s vault
```

Comment:
There's another change in `test-requirements.txt`:

```diff
@ test-requirements.txt:2 @
molecule==2.16.0
-docker-py-1.10.6
+docker-py==1.10.6
```

Reply:
I should have tested that myself, good catch!